src/Transports.AspNetCore.NewtonsoftJson/GraphQLRequestDeserializer.cs

```diff
@@ -0,0 +1,69 @@
+using GraphQL.Server.Transports.AspNetCore.Common;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using GraphQLRequestBase = GraphQL.Server.Common.GraphQLRequest;
+
+namespace GraphQL.Server.Transports.AspNetCore.NewtonsoftJson
+{
+    /// <summary>
+    /// Implementation of an <see cref="IGraphQLRequestDeserializer"/> that uses Newtonsoft.Json.
+    /// </summary>
+    public class GraphQLRequestDeserializer : IGraphQLRequestDeserializer
+    {
+        private readonly JsonSerializer _serializer;
+
+        public GraphQLRequestDeserializer(Action<JsonSerializerSettings> configure)
+        {
+            var settings = new JsonSerializerSettings();
+            configure(settings);
```

Comment:
Consider that configure may be null, also in S.T.J.

Reply:
Now null check before invoking.